packages/fxa-payments-server/src/components/NewUserEmailForm/en.ftl

```diff
@@ -8,7 +8,10 @@ new-user-enter-email =
   .label = Enter your email
 new-user-confirm-email =
   .label = Confirm your email
-new-user-subscribe-product-updates = I’d like to receive product updates from { -brand-name-firefox }
+new-user-subscribe-product-updates-mozilla = I’d like to receive product news and updates from { -brand-mozilla }
+new-user-subscribe-product-updates-snp = I’d like to receive security and privacy news and updates from { -brand-mozilla }
```

Comment:
Nit: Add a comma - "security and privacy news, and updates from"

Reply:
Discussed this and decided to leave as is for now.